pkg/apis/radix/v1/radixapptypes.go

```diff
@@ -945,45 +945,50 @@ type RadixVolumeMount struct {
 	Path string `json:"path"` // Path within the pod (replica), where the volume mount has been mounted to
 
 	// GID defines the group ID (number) which will be set as owner of the mounted volume.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
 	GID string `json:"gid,omitempty"` // Optional. Volume mount owner GroupID. Used when drivers do not honor fsGroup securityContext setting. https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/docs/driver-parameters.md
 
 	// UID defines the user ID (number) which will be set as owner of the mounted volume.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
 	UID string `json:"uid,omitempty"` // Optional. Volume mount owner UserID. Used instead of GID.
 
 	// TODO: describe
 	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
```

Comment:
```suggestion
	// Deprecated: use BlobFuse2 instead.
	// TODO: describe
	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
	// +optional
```
The first comment must start with "Deprecated: " for field to be flagged as deprecated

Reply:
Fixed